tests/units/anta_tests/test_connectivity.py

```diff
@@ -256,16 +250,17 @@
                 },
             },
         ],
+        "inputs": {
+            "neighbors": [
+                {"port": "Ethernet1", "neighbor_device": "DC1-SPINE1", "neighbor_port": "Ethernet1"},
+                {"port": "Ethernet2", "neighbor_device": "DC1-SPINE2", "neighbor_port": "Ethernet1"},
+            ],
+        },
```

Comment:
Why were these reordered? Just curious.

Reply:
Just to be consistent.
We have inputs after the eos_data all the places.